activejob/test/cases/test_helper_test.rb

```diff
@@ -1197,7 +1185,7 @@ def test_assert_performed_jobs_without_block_with_except_and_queue_options
     assert_performed_jobs 1, except: HelloJob, queue: :other_queue
   end
 
-  def test_assert_performed_jobs_with_except_and_queue_options_failuree
+  def test_assert_performed_jobs_with_except_and_queue_options_failure
```

Comment:
could you please just change the name to `test_assert_performed_jobs_without_block_with_except_and_queue_options_failure`

Reply:
Ah.... yes. Fixed.